examples/flow.cpp

```diff
@@ -251,15 +251,31 @@ try
     const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
     Opm::BlackoilOutputWriter outputWriter(grid, param, eclipseState, pu );
 
+    int numCells = Opm::UgGridHelpers::numCells(grid);
```

Comment:
I think the best solution would be that a grid always provides a compressed -> Cartesian cell mapping. but well, I'll add a free function to do this in the afternoon.